examples/guides/se-restful-webservice/README.adoc

```diff
@@ -40,19 +43,19 @@ You'll learn how to use Helidon quickly to create a RESTful web service that acc
 |`PUT localhost:8080/greet/greeting/Hola` |Changes the greeting used in subsequent responses
 |===
 
-We create the app in three main steps:
+Create the app in three main steps:
```

Comment:
"You'll create the app..." would be better, consistent with the "You'll learn..." intro to the previous section.

Reply:
Done